### DISCUSSION TITLE
How do I make the __test_project for the pytest tests stay around so that I can browse it when debugging?

### DISCUSSION THREAD

**Discussion by Quincunx271** [vector-of-bool/dds#79] [Q&A] [answered]
I've often found that I want to look closer at the generated `__test_project`s from `ProjectOpener.open` when debugging a test, but there are finalizers added to remove the directory. Is there a good way to disable this? Is commenting out the finalizers the best I can do?

**Comment by vector-of-bool** [comment, marked as answer]
At the moment, yeah, that's the best that can be done. If you are debugging the Python test directly, the `__test_project` will stay around while you step through the test, otherwise it will be deleted immediately.

**Reply by Quincunx271** [reply]
Well, debugging is probably nicer than what I was doing. I keep forgetting that Python has a debugger too :D
